build/configs/rtl8721csm/loadable_apps/defconfig

```diff
@@ -247,9 +247,9 @@ CONFIG_ARCH_BOARD_HAVE_FLASH=y
 #
 CONFIG_FLASH_PARTITION=y
 CONFIG_FLASH_MINOR=0
-CONFIG_FLASH_PART_SIZE="16,8,1024,1024,5608,"
-CONFIG_FLASH_PART_TYPE="none,none,kernel,kernel,smartfs,"
-CONFIG_FLASH_PART_NAME="km0_bl,km4_bl,kernel,kernel,userfs,"
+CONFIG_FLASH_PART_SIZE="16,8,16,472,1024,1024,5608,"
+CONFIG_FLASH_PART_TYPE="none,none,none,none,kernel,kernel,smartfs,"
```

Comment:
If you want to add more partitions, you should change the number of `CONFIG_AMEBAD_AUTOMOUNT_USERFS_DEVNAME` also.
Please change `/dev/smart0p4` to `"/dev/smart0p6`.